src/main/java/de/plushnikov/intellij/plugin/quickfix/LombokSneakyThrowsQuickFix.java

```diff
@@ -0,0 +1,96 @@
+package de.plushnikov.intellij.plugin.quickfix;
+
+import com.intellij.codeInsight.intention.IntentionAction;
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.JavaPsiFacade;
+import com.intellij.psi.PsiAnnotation;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiClassInitializer;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementFactory;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiJavaFile;
+import com.intellij.psi.PsiLambdaExpression;
+import com.intellij.psi.PsiMethod;
+import com.intellij.psi.PsiMethodReferenceExpression;
+import com.intellij.psi.codeStyle.JavaCodeStyleManager;
+import com.intellij.psi.search.GlobalSearchScope;
+import com.intellij.psi.util.PsiTreeUtil;
+import com.intellij.util.IncorrectOperationException;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import lombok.SneakyThrows;
+import de.plushnikov.intellij.plugin.util.PsiAnnotationSearchUtil;
+
+
+public class LombokSneakyThrowsQuickFix extends LocalQuickFixOnPsiElement implements IntentionAction {
```

Comment:
I think you don't need to write new class for annotation-creation. Simply use existing com.intellij.codeInsight.intention.AddAnnotationFix class.

Reply:
Thanks, I also think it will be better. Less code is fewer bugs.